pkgs/development/python-modules/phik/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPy3k,
+  nbconvert,
+  pytest-pylint,
+  numba,
+  pandas,
+  matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "phik";
+  version = "0.9.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    format = "wheel";
+    python = "py3";
+    abi = "none";
+    platform = "any";
+    sha256 = "c398452c5c1eea153905666b289c6a153712cf3d58811fa41e2bbbd27a65d678";
+  };
+
+  propagatedBuildInputs = [
+    nbconvert
+    pytest-pylint
+    numba
+    pandas
+    matplotlib
```

Comment:
I think `numpy` and `scipy` ought to be added as `propagatedBuildInputs` too: https://github.com/KaveIO/PhiK/blob/v0.9.8/setup.py#L39-L45. They are probably unnecessary as a result of including `pandas` or something, and I'm not sure if there's a policy on this, but it seems sensible.